schemas/roofing_tiles_slate/v2.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "v2.json",
+  "title": "SDS schema for the Roofing Tiles + Slate surveys",
+  "type": "object",
+  "properties": {
+    "schema_version": {
+      "const": "v2",
+      "description": "Version of the schema spec"
+    },
+    "survey_id": {
+      "type": "string",
+      "description": "The identifier for a particular survey",
+      "enum": ["068", "071"]
```

Comment:
The survey_id is an Enum that can have more than 1 value (2 values found for rooting tiles slate and sand and gravel). We have asked them to duplicate into two files in separate folder, each with 1 survey_id last time.

Reply:
From our understanding, this was a piece of work that came out from a different meeting, so this will be addressed in a future ticket